stimuli_ci_generation/src/server/datastore.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+"""
+This module contains helper methods to modify firestore documents
+"""
+
+from google.cloud import datastore
+
+db = datastore.Client()
+
+
+def update_user_doc(participant_id, experiment_id, stimuli_status=None):
+    """
+    Update a firestore user document with corresponding attributes
+    Args:
+        participant_id: user's uid
+        experiment_id: experiment's id
+        stimuli_status: 'completed' or None
+    Returns:
+        None
+    """
+    user_doc_ref = db.collection("Users").document(participant_id)
+    user_doc_ref.set({"sie-stimuli-generation-status": stimuli_status or ""})
```

Comment:
Oh could you use sie_stimuli_generation_status for the field key name? this way I can easily access it in json object.

Reply:
sure!